### ISSUE TITLE
RecursionError: maximum recursion depth exceeded

### ISSUE THREAD

**Issue by cheuel-novero** [mvantellingen/python-zeep#19]
If I try a 
```python3 -mzeep http://almdemo.polarion.com/polarion/ws/services/TrackerWebService?wsdl```

with the latest master version zeep fails with
```
...
  File "/home/cheuel/devel/external/python-zeep/src/zeep/xsd/visitor.py", line 116, in visit_import
    schema_node, self.schema._transport, location, self.parser_context)
  File "/home/cheuel/devel/external/python-zeep/src/zeep/xsd/schema.py", line 45, in __init__
    visitor.visit_schema(node)
  File "/home/cheuel/devel/external/python-zeep/src/zeep/xsd/visitor.py", line 82, in visit_schema
    self.process(node, parent=parent)
  File "/home/cheuel/devel/external/python-zeep/src/zeep/xsd/visitor.py", line 40, in process
    result = visit_func(self, node, parent)
  File "/home/cheuel/devel/external/python-zeep/src/zeep/xsd/visitor.py", line 100, in visit_import
    location = absolute_location(location, self.schema._location)
  File "/home/cheuel/devel/external/python-zeep/src/zeep/parser.py", line 50, in absolute_location
    if urlparse(location).scheme:
  File "/usr/lib/python3.5/urllib/parse.py", line 294, in urlparse
    url, scheme, _coerce_result = _coerce_args(url, scheme)
RecursionError: maximum recursion depth exceeded
```

**Comment by cheuel-novero** [comment]
git bisect seems to blame this commit f1eef47 for causing the RecursionError:
```
f1eef4711fedd803fa529ee38b18fe5914c92a8d is the first bad commit
commit f1eef4711fedd803fa529ee38b18fe5914c92a8d
Author: Michael van Tellingen <michael@mvantellingen.nl>
Date:   Wed May 11 23:02:05 2016 +0200

    Set original location value on Schema objects when they are internal

:040000 040000 cbb8cc2dbb89ef1f45f01042b1a2bfb9dee16fe6 6a21a56196384d43237e14afa0f36d9c9881b945 M      src
```

**Comment by mvantellingen** [comment]
Oops, sorry :-) Should be fixed in b1c2e08520f011f26bbee283da10231646cc3a48

**Comment by cheuel-novero** [comment]
Yes, that fixed it! Thanks!
